### PR TITLE
Feature scheduler with test duration

### PR DESCRIPTION
**Pull request by eemrdog**

### CODE REVIEW

**Review by tobigremmer-dt — DISMISSED** [review]
Looks good!

File: README.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ Library versions are listed below:
 | --------------- | ------- |
 | v1.0 | Initial Release |
 | v1.1 | Support for new runner and dynamic jmeter path |
+| v1.2.1 | Support for new jmeter runner and dynamic jmeter install path |
+| v1.3 | Support for jmeter scheduler capability by setting the test duration |
```
Comment (tobigremmer-dt) [inline README.md]: Please use a semver compliant version: https://semver.org

Comment (eemrdog) [reply README.md]: done

**Review by tobigremmer-dt — APPROVED** [review]
Looks good!

**Pull request merged**